app/src/main/res/drawable/ic_info_outline_black_24dp.xml

```diff
@@ -4,6 +4,6 @@
         android:viewportWidth="24.0"
         android:viewportHeight="24.0">
     <path
-        android:fillColor="#FF000000"
+        android:fillColor="#FFFFFF"
```

Comment:
Undo unrelated changes

Reply:
I am new in open source contribution. How can I remove this commit?

Reply:
You can revert a commit by using `git revert <commit sha>` or else you can manually undo the changes and commit it. :)

Reply:
I am creating a new pull request by doing fresh commit

Reply:
Sure. Please close this pull request once your new PR is up. :)

Reply:
yea sure